### PR TITLE
Warning for rh-php71 obsolete

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6116

A warning box is displayed  when rh-php71 is enabled for a  virtualhost, the dropdown phpVersion displays the value `php71` only if used by the vhost, else only 'default', 'php72' and  'php73' are visible

### CODE REVIEW

**Review by DavidePrincipi — COMMENTED** [review]
I'd mark 7.1 as "obsolete" also in line R103

https://github.com/NethServer/nethserver-httpd/pull/77/files#diff-e0a61132689ec28296809b61bb6db5f0R103

Could you attach a screenshot of the result?

**Comment by stephdl** [comment]
On an existing vhost with php71

Screenshot (30)

**Comment by stephdl** [comment]
If you create a new vhost, php71 is no more available (a smart guy could create it with the command line)


Capture d’écran de 2020-04-16 00-12-29

**Comment by DavidePrincipi** [comment]
Ok, when I create a new vhost, what is the default selected entry? 5.4?

**Comment by stephdl** [comment]
we strongly encourage sysadmin to use the default, so yes, when you create a vhost, default (php54) is set.

Like this we do not change the code behind, we will clean it maybe after, but we simply hide php71 for new vhost or when the php version is not set to php71.

So for an an existing Vhost set to php54 or php72,php73 you cannot see php71 option, for a previous vhost set to php71, once you have changed the php version, you cannot see anymore php71. Only the CLI could help you to go back to php71

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -120,7 +120,9 @@
     "php_version_php71": "PHP 7.1",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]:
```suggestion
    "php_version_php71": "PHP 7.1 (obsolete)",
```

**Review by gsanchietti — COMMENTED** [review]
For me it's good, I suggest just only a minor label change.

**Comment by stephdl** [comment]
@gsanchietti what about to display also in the vhost list ?

Screenshot (33)

Screenshot (34)

File: ui/public/i18n/language.json (hunk 1 above)
Comment (stephdl) [reply ui/public/i18n/language.json]: done

**Review by gsanchietti — APPROVED** [review]

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**